tests/peregrine.py

```diff
@@ -19,8 +19,8 @@
 
 from reV import TESTDATADIR
 from reV.handlers.outputs import Outputs
-from reV.utilities.execution import PBS
-from reV.utilities.loggers import init_logger
+from rex.utilities.execution import PBS
```

Comment:
We might just want to remove this file (peregrine.py). We're definitely not maintaining it.

Reply:
love it, gone!